src/compositions/useBlockDocument.ts

```diff
@@ -13,6 +13,7 @@ export function useBlockDocument(blockDocumentId: MaybeRefOrGetter<string | null
   const can = useCan()
 
   const getter: Getter<[string] | null> = () => {
+    // todo: audit [cchoy]
```

Comment:
I'm not sure what these checks are necessary for 🤔 shouldn't we just let the backend handle this check.

hmmmm either way all of these can.read.* checks in compositions should be _okay_ since you must have can.read in order to get any escalated priveleges to the objects via acl anyways.

Reply:
mm to that extent, I'm going to remove these

Reply:
These were here to just avoid making a call we knew was going to fail. 

Since that's no longer the case it might make sense to remove these permission checks in compositions that fetch objects with acls.